deployment/terraform/assets/cluster.tf

```diff
@@ -82,6 +82,47 @@ resource "aws_instance" "app_server" {
   }
 }
 
+data "aws_vpc" "selected" {
+  tags = {
+    Name = "Default VPC"
+  }
+}
+
+data "aws_subnets" "loadtest_subnets" {
+  filter {
+    name = "vpc-id"
+    values = [data.aws_vpc.selected.id]
+  }
+
+  tags = {
+    Name = "loadtest*"
+  }
+}
+
+resource "aws_efs_file_system" "efs_shared" {
+  count = var.create_efs ? 1 : 0
+
+  tags = {
+   Name = "Shared-fs-${var.cluster_name}"
+  }
+}
+
+resource "aws_efs_mount_target" "efs_mount" {
+  for_each = (var.create_efs ? toset(data.aws_subnets.loadtest_subnets.ids) : toset({}))
+  file_system_id = aws_efs_file_system.efs_shared.0.id
+  subnet_id      = each.value
+  security_groups = [aws_security_group.app[0].id]
+}
+
+resource "aws_efs_access_point" "shared_dir" {
+  count = var.create_efs ? 1 : 0
+
+  file_system_id = aws_efs_file_system.efs_shared.0.id
+
+  root_directory {
+    path = "/"
+  }
+}
```

Comment:
Can we also add tag names to these in the form of `"${var.cluster_name}-whatever-whatever"`?

Reply:
We can add it to `aws_efs_access_point` but can't add it to `aws_efs_mount_target`, I'll go ahead and add only to the access point.